app/src/main/res/values/strings.xml

```diff
@@ -40,6 +40,7 @@
     <!-- Snackbar Strings -->
     <string name="sb_ok">OKAY</string>
     <string name="sb_permission_storage_rationale">Storage permission needed to upload file to Wetfish.</string>
+    <string name="sb_permission_storage_rationale_export_import">Storage permission needed to be able to export and import the Wetfish DB.</string>
```

Comment:
I don't think anybody is going to know what "the wetfish DB" means. Maybe it would be better to say "Storage permission needed to be able to export and import your photo gallery"?